src/services/ui/src/features/package-actions/lib/modules/toggle-rai-withdraw/index.tsx

```diff
@@ -46,3 +47,9 @@ export const defaultDisableRaiWithdrawContent: FormContentHydrator = (
     body: "The state will be able to withdraw its RAI response. It may take up to a minute for this change to be applied.",
   },
 });
+
+export const raiWithdrawalDisabled: CheckStatusFunction = (checks) =>
+  !checks.hasEnabledRaiWithdraw;
+
+export const raiWithdrawalEnabled: CheckStatusFunction = (checks) =>
+  checks.hasEnabledRaiWithdraw;
```

Comment:
these can just be one function and measured with null assertions: `!raiWithdrawEnabled`

Reply:
Good point. I can definitely rename

Reply:
Discussed further together and because of the difficulty of what it would take we won't be doing it. Note for the future, negating a function definition returns a boolean.